### PR TITLE
Default to Next CFP closing date and add sort by Next CFP start data

### PR DESCRIPTION
**Pull request by josecanhelp**
image

Resolves Issue #16

### CODE REVIEW

File: app/Http/Controllers/ConferencesController.php [hunk 1]
```diff
@@ -43,92 +43,87 @@ public function index(Request $request)
...
-
+        
         switch ($request->input('sort')) {
             case 'date':
-                $conferences = $conferences->sortBy(function (Conference $model) {
-                    return $model->starts_at;
-                });
+                $conferences = $conferences->sortBy->starts_at;
                 break;
             case 'closing_next':
```
[the remark this answers is not in the document]
Comment (josecanhelp) [reply app/Http/Controllers/ConferencesController.php]: They are the same. `sortBy` is an available method for higher order messages.

[the remark this answers is not in the document]
Comment (mattstauffer) [reply app/Http/Controllers/ConferencesController.php]: No, I mean closing next and whatever was after it that is passing through

**Comment by mattstauffer** [comment]
@josecanhelp no rush but i just found this--would you take a look at it just when you have free some time? <3

File: app/Http/Controllers/ConferencesController.php (hunk 1 above)
[the remark this answers is not in the document]
Comment (josecanhelp) [reply app/Http/Controllers/ConferencesController.php]: Ah, I see, yes. `closing_next` and `opening_next` are different, but use the same switch case here. I wonder why this was done. I'm looking into it now.

[the remark this answers is not in the document]
Comment (josecanhelp) [reply app/Http/Controllers/ConferencesController.php]: The default case is `closing_next` so the fix was to move that case statement right before the default case.

**Comment by josecanhelp** [comment]
@mattstauffer Was there anything else we needed to check on this PR?

File: app/Http/Controllers/ConferencesController.php [hunk 2]
```diff
@@ -46,28 +46,23 @@ public function index(Request $request)
 
         switch ($request->input('sort')) {
             case 'date':
+                $conferences = $conferences->sortBy->starts_at;
+                break;
+            case 'opening_next':
+                // Forces closed CFPs to the end.
```
Comment (mattstauffer) [inline app/Http/Controllers/ConferencesController.php]: I think this comment should be "Force CFPs with no CFP start date to the end"

File: resources/views/account/public-profile/index.blade.php [hunk 3]
```diff
@@ -2,43 +2,38 @@
...
 @section('content')
 
-    <div class="container">
-        <ol class="breadcrumb">
-            <li><a href="/">Home</a></li>
-            <li class="active"><a href="{{ route('speakers-public.index') }}">Speakers</a></li>
-        </ol>
+    <div class="container body">
+        <div class="col-md-10 col-md-push-1">
+            <h1>Speaker Profiles</h1>
```
Comment (mattstauffer) [inline resources/views/account/public-profile/index.blade.php]: Was this breadcrumb bad or is this a merge wonky-ness?

File: routes/web.php [hunk 4]
```diff
@@ -89,28 +89,6 @@
 Route::get('conferences', ['as' => 'conferences.index', 'uses' => 'ConferencesController@index']);
 Route::get('conferences/{id}', ['as' => 'conferences.show', 'uses' => 'ConferencesController@show']);
```
Comment (mattstauffer) [inline routes/web.php]: Are these not in use anymore? I remember something happening with these but surprised to see in this PR

File: resources/views/account/public-profile/index.blade.php (hunk 3 above)
Comment (josecanhelp) [reply resources/views/account/public-profile/index.blade.php]: This looks like a merge issue. I'll check it out.

**Pull request closed without merging**